release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3362,6 +3362,59 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, during {product-title} software-defined networking (SDN) to `OVNKubernetes` migration with NMState managing `br-ex`, the `mtu-migration`` service that was active before `wait-for-primary-ip` caused a maximum transmission unit (MTU) migration failure. This action resulted in network connectivity issues for the user after migration to {product-version} 4.16. With this release, the services in NMState configurations are reordered, placing 'mtu-migration' after `wait-for-primary-ip`. This arrangement improves stability during migration, and prevents a `mtu-migration` service failure after the first reboot. (link:https://issues.redhat.com/browse/OCPBUGS-54831[*OCPBUGS-54831*])
+
+* Previously, containers that used the SELinux `container_logreader_t` domain to view container logs on a host at `/var/log` could not access logs in the `/var/log/containers` subdirectory. This issue occurred because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` and containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54818[OCPBUGS-54818])
+
+* Previously, a bug was encountered during limited live migration, that was caused by an incomplete container network interface (CNI) redeployment. This issue led to potential network connectivity problems during rollback to {product-title} software-defined networking (SDN) for users. With this release, a bug fix is made, adding an annotation for the node's primary IP address to handle non-cloud platforms during egress IP address rollback. The fix ensures a smooth rollback to {product-title} SDN, resolving stuck issues that occur during limited live migration, and improving network connectivity. (link:https://issues.redhat.com/browse/OCPBUGS-53317[OCPBUGS-53317])
+
+* Previously, a specific commit caused MAC binding flows to reach the configured maximum age and were removed in {product-title} {product-version} 4.14.48. This action caused data plane traffic interruptions when the OVN Controller was slow to update address sets. With this release, the issue in the OVN MAC binding refresh on {azure-first} is corrected. (link:https://issues.redhat.com/browse/OCPBUGS-53151[OCPBUGS-53151])
+
+*  Previously, a User Datagram Protocol (UDP) packet that was larger than the maximum transmission unit (MTU) value that was set for the cluster could not be sent to the packet endpoint by using a service. With this release, the pod IP address is used instead of the service IP address regardless of the packet size, and the UDP packet is sent to the endpoint. (link:https://issues.redhat.com/browse/OCPBUGS-50581[OCPBUGS-50581])
```

Comment:
```suggestion
* Previously, a User Datagram Protocol (UDP) packet that was larger than the maximum transmission unit (MTU) value that was set for the cluster could not be sent to the packet endpoint by using a service. With this release, the pod IP address is used instead of the service IP address regardless of the packet size, and the UDP packet is sent to the endpoint. (link:https://issues.redhat.com/browse/OCPBUGS-50581[OCPBUGS-50581])
```

- nit: extra space